completions/mfiutil

```diff
@@ -0,0 +1,126 @@
+# mfiutil completion                                       -*- shell-script -*-
+
+[[ $OSTYPE == *@(freebsd|dragonflybsd|darwin|linux|solaris)* ]] || return 1
+
+_comp_cmd_mfiutil()
+{
+    local cur prev words cword comp_args
+    _comp_initialize -- "$@" || return
+
+    if [[ $cur == -* ]]; then
+        # Complete global options
+        local options="-u -d -e"
+        case "$OSTYPE" in
+            *freebsd*)
+                options="$options -D -t"
```

Comment:
```suggestion
                options+=" -D -t"
```
...and similarly below. I suppose we could use an array for constructing the set of options instead of a string, but that's no biggie.

Reply:
cbd266155020194ec824028fb5c0d7ac95c89474